tools/build_chain.sh

```diff
@@ -502,6 +510,7 @@ generate_config_ini()
     flush=${auto_flush}
     log_flush_threshold=100
 EOF
+    printf "  [%d] p2p:%-5d  channel:%-5d  jsonrpc:%-5d\n" ${node_index} $(( offset + port_array[0] )) $(( offset + port_array[1] )) $(( offset + port_array[2] )) >>${logfile}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5311491)